src/ophys_etl/pipelines/suite2p_registration.py

```diff
@@ -23,6 +26,11 @@ class Suite2PRegistrationInputSchema(argschema.ArgSchema):
         required=True,
         description=("hdf5 format of diagnostics harvested from Suite2P "
                      "ops.npy"))
+    motion_offset_output = argschema.fields.OutputFile(
```

Comment:
don't we want this to just supersede `motion_diagnostics_output` ?
I.e.  just ditch the hdf5 at this point.
If we keep the same key, Wayne won't need to change the strategy. But, probably not too late, either.